README.md

```diff
@@ -1,8 +1,8 @@
-[![](https://raw.githubusercontent.com/mapbox/mapbox-java/master/.github/splash-img.png)](https://www.mapbox.com/android-docs/mapbox-services/)
```

Comment:
Yeah, I liked the idea of a splash image. The current one, without nav and without "much more". I'd add instead "RxJava support" and "static maps". Also, I'd put the badges before the splash image.